### ISSUE TITLE
Use fee cut instead of fee share in CLI

### ISSUE THREAD

**Issue by kyriediculous** [livepeer/go-livepeer#1808]
**Is your feature request related to a problem? Please describe.**
The explorer uses fee cut as user facing language even though the protocol uses feeShare (the opposite). 

It would be good that all user facing language is ubiquitous to avoid confusion. 

From discord: 

> AcesHigh — Today at 2:39 AM
I noticed that he explorer shows "Fee Cut" as a column and the command line has a parameter feeShare.  Does feeShare mean the opposite of feeCut - i.e. feeSHare is how much fees the orchestrator will share back to the delegators and feeCut is how much the orchestrator keeps??
In other words, if I set feeShare to 60, Fee Cut will show as 40 on the UI?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
